### ISSUE TITLE
Include Home Coach sensor-data?

### ISSUE THREAD

**Issue by arnepluhar** [xperimental/netatmo-exporter#43]
Besides the Netatmo Energy, I somehow ended up buying a Home Coach sensor, as I thought it might integrate into the Weather Station interface somehow. It didn't. The documentation is great, though. Can I somehow help integrating this, too?
https://github.com/labordep/PharoNetatmoAPI

**Comment by xperimental** [comment]
Hi @arnepluhar ,

I'm sorry, but I'm not really sure what you're trying to achieve.

I'm not familiar with the "Home Coach sensor" and, currently, the `netatmo-exporter` has no support for the "Netatmo energy" products. I'm not opposed to extending support to other Netatmo APIs such as the energy one, but I don't know if adding support for a completely different product ("Home Coach") is something that should be integrated into the "netatmo exporter".

Or am I misunderstanding the situation and "Home Coach" is a Netatmo product?

**Comment by aexvir** [comment]
@xperimental yep, [home coach](https://shop.netatmo.com/en-eu/aircare/home-coach/homecoach) is an indoor sensor from netatmo, it's exposed via their `aircare` api
https://dev.netatmo.com/apidocumentation/aircare#gethomecoachsdata

I'm also looking forward to this, and I'd be up for contributing if you don't have the time to add support for it

**Comment by xperimental** [comment]
Seeing as all three open issues are requests for integrating more of NetAtmo's products into the exporter, I think I need to give this a look. Can't give any guarantees, but I hope to do some hobby project work again soon :slightly_smiling_face:

**Comment by neothematrix** [comment]
thanks @xperimental 
if it could be useful, I've managed to fork your project and modify it to handle homecoach, but I'm not skilled enough to handle both devices, so my fork will just work with homecoach: neothematrix/netatmo-exporter

**Comment by xperimental** [comment]
@neothematrix I had seen the forks already, but had not looked into the code yet. Good if this works for you, but it still needs some work to be integrated into the existing codebase without replacing the API for the other sensors.
